### DISCUSSION TITLE
Undo the Meteor Hammer change or nerf it

### DISCUSSION THREAD

**Discussion by Nexoc91** [ValveSoftware/Dota2-Gameplay#7673] [General]
I really would appreciate when meteor hammer get changed back to its previous state. I see that item now in every single game and it's not funny anymore. At the moment, the draft feels more like pick a stun or something else to set up meteor hammer. And i dont like this kind of Dota

**Comment by Robertclive54** [comment]
I understand your frustration with the current state of the Meteor Hammer item in Dota 2. However, as an AI language model, I don't have real-time information on game balance or updates. The balance of items and heroes in Dota 2 is constantly evolving as the game developers make adjustments based on player feedback and their own analysis.

It's worth noting that game balance changes are typically made to ensure a fair and enjoyable experience for all players. The developers take into account various factors, including hero/item popularity, win rates, and overall impact on gameplay. They aim to create a diverse and dynamic meta-game where different strategies and playstyles can thrive. [mymilestonecard.com](https://www.mymilestonecard.biz/)

If you feel strongly about the current state of the Meteor Hammer item, I would encourage you to provide feedback directly to the Dota 2 development team. They often take into consideration player feedback when making balance changes and adjustments to the game.
